packages/core/src/test/node/spec/conversation/ConversationService.test.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+/* eslint-disable no-magic-numbers */
+const APIClient = require('@wireapp/api-client');
+const UUID = require('pure-uuid');
+const {Account} = require('@wireapp/core');
+const {Config} = require('@wireapp/api-client/dist/commonjs/Config');
+const {MemoryEngine} = require('@wireapp/store-engine');
+
+const createMessage = (conversationRepository, content) => {
+  const customTextMessage = conversationRepository.protocolBuffers.GenericMessage.create({
+    messageId: new UUID(4).format(),
+    text: conversationRepository.protocolBuffers.Text.create({content}),
+  });
+
+  return conversationRepository.protocolBuffers.GenericMessage.encode(customTextMessage).finish();
+};
+
+const generatePreKeyBundles = (users, devicesPerUser) => {
+  const preKeyBundles = {};
+
+  new Array(users)
+    .fill()
+    .map(() => new UUID(4).format())
+    .forEach(userId => {
+      preKeyBundles[userId] = {};
+      new Array(devicesPerUser)
+        .fill()
+        .map(() => new UUID(4).format())
+        .forEach(deviceId => (preKeyBundles[userId][deviceId] = {}));
+    });
+
+  return preKeyBundles;
+};
+
+describe('ConversationService', () => {
+  let account;
+
+  beforeAll(async done => {
+    const engine = new MemoryEngine();
+    await engine.init('');
+
+    const client = new APIClient(new Config(engine, APIClient.BACKEND.STAGING));
```

Comment:
In the test we can init our API client with an object: `new APIClient({store: engine, urls: APIClient.BACKEND.STAGING);`

We should use the object initialization because on the long run we want to get rid off `new Config()`.